build-pkg.sh

```diff
@@ -36,7 +36,7 @@ fpm -s dir -n honeytail \
     -t $pkg_type \
     -a $arch \
     --pre-install=./preinstall \
-    $GOPATH/bin/honeytail-linux-${arch}=/usr/bin/honeytail \
+    ./artifacts/honeytail-linux-${arch}=/usr/bin/honeytail \
```

Comment:
We only use the build script in CI, right? Changing the directory to artifacts will make it break locally, but I don't know if we care, unless we build the packages locally.

Reply:
It's my understanding that CI is the only thing using this script. Certainly it's the only automation in this repo that uses it. The original $GOPATH/bin path is also in test.sh, but CI doesn't appear to use that.

Reply:
Update: I was wrong about `./artifacts`. All the jobs putting things in `~/artifacts` ... the default working directory for a job is `~/project`. So, yea, this broke.